lib/puppet/parser/functions/foreman.rb

```diff
@@ -36,6 +36,7 @@
 #
 # Happy Foreman API-ing!
 
+require "pry"
```

Comment:
What is the purpose of this line anyway? IMHO it's simply not possible to *require* a debugger gem for normal production installs (AFAICT that would even needed to get installed in the JRuby Puppetserver if running on that).

The fix for the problem is the increase of the timeout, no?